lib/NMISNG/Node.pm

```diff
@@ -1761,6 +1761,36 @@ sub validate
 		}
 	}
 
+	# check for a validation plugin
+	# run if present else do nothing.
+	$self->nmisng->log->info("Initializing validation plugins");
+	
+	my @plugins = $self->nmisng->validation_plugins;
+	$self->nmisng->log->info("Plugins are -".Dumper(\@plugins));
```

Comment:
This needs to be quieter

It also needs to wrap the inner message in sub{} so that it is not executed unless that level is active